tests/ovs_test.go

```diff
@@ -32,4 +35,42 @@ var _ = Describe("ovs-cni", func() {
 			})
 		})
 	})
+
+	Describe("pod ovs-bridge connectivity tests", func() {
+		bridgeName := "br-test"
+		Context("When creating ovs-bridge on the host node", func() {
+			BeforeEach( func() {
+				removeOvsBridgeOnNode(bridgeName)
+				addOvsBridgeOnNode(bridgeName)
+			})
+			AfterEach( func() {
+				removeOvsBridgeOnNode(bridgeName)
+			})
+
+			Context("and adding ovs network with vlan", func() {
+				nadName := "ovs-net"
+				BeforeEach( func() {
+					createNetworkAttachmentDefinition(nadName, bridgeName, `{ "cniVersion": "0.3.1", "type": "ovs", "bridge": "` + bridgeName + `", "vlan": 100 }`)
+				})
+				AfterEach( func() {
+					removeNetworkAttachmentDefinition(nadName)
+					deletePodsInTestNamespace()
+				})
+
+				It("Should enable connectivity between two pods connected to the same bridge", func() {
+					cidrPod1 := "11.0.0.1/24"
+					cidrPod2 := "11.0.0.2/24"
+					createPrivilegedPodWithIp("pod-test-1", nadName, cidrPod1)
```

Comment:
Why do we need to create privileged pod for pinging, non-privileged one wont be enough?

Reply:
unfortunately not, since we need to manually change the pod's ip (ovs-cni configuration does not currently handle ipam configuration).

Reply:
Why is it necessary to set specific IP address for the pod?

Reply:
simply easier to ping if you know the ips..